File: docs-old/datatools/codesnippet/CSharp/insert-new-records-into-a-database_1.cs
            // Create a new row.
            NorthwindDataSet.RegionRow newRegionRow;
            newRegionRow = northwindDataSet.Region.NewRegionRow();
            newRegionRow.RegionID = 5;
            newRegionRow.RegionDescription = "NorthWestern";

            // Add the row to the Region table
            this.northwindDataSet.Region.Rows.Add(newRegionRow);

            // Save the new row to the database
            this.regionTableAdapter.Update(this.northwindDataSet.Region);

File: docs-old-1103-2/data-tools/codesnippet/VisualBasic/insert-new-records-into-a-database_1.vb
        ' Create a new row.
        Dim newRegionRow As NorthwindDataSet.RegionRow
        newRegionRow = Me.NorthwindDataSet._Region.NewRegionRow()
        newRegionRow.RegionID = 5
        newRegionRow.RegionDescription = "NorthWestern"

        ' Add the row to the Region table
        Me.NorthwindDataSet._Region.Rows.Add(newRegionRow)

        ' Save the new row to the database
        Me.RegionTableAdapter.Update(Me.NorthwindDataSet._Region)